js/popover.js

```diff
@@ -64,9 +64,22 @@
   Popover.prototype.getContent = function () {
     var $e = this.$element
     var o  = this.options
+    var contentType = typeof o.content
+
+    if (!/string|function/.test(contentType)) {
+      throw new Error('Popover received type ${' + contentType + '}. Was expecting type string or a function.')
```

Comment:
Oops… Sorry! I admit I need to read more about ES6. Thanks for clearing that up. :flushed: